test/powershell/Modules/PackageManagement/PackageManagement.Tests.ps1

```diff
@@ -19,13 +19,24 @@ $InternalSource = 'OneGetTestSource'
 Describe "PackageManagement Acceptance Test" -Tags "Feature" {
 
  BeforeAll{
-    Register-PackageSource -Name Nugettest -provider NuGet -Location https://www.nuget.org/api/v2 -Force
-    Register-PackageSource -Name $InternalSource -Location $InternalGallery -ProviderName 'PowerShellGet' -Trusted -ErrorAction SilentlyContinue
+
+    # Setting all It block to pending as the test fail due to https://github.com/dotnet/runtime/issues/65013
```

Comment:
The issue was fixed in .Net but we get this 2-3 month later so I think we need a tracking issue in PowerShell repo.

Reply:
@adityapatwardhan If the fix is not included in preview 1, are we still going to move to 7.0 preview 1?
I asked the question here: https://github.com/dotnet/runtime/pull/65157#issuecomment-1039617623

Reply:
Got [the confirmation](https://github.com/dotnet/runtime/pull/65157#issuecomment-1039630436) that this fix won't be in 7.0 preview 1. Given that, I think we should skip the 7.0 preview 1. Thoughts?

Reply:
I have confirmed with Steve to that we could ship using .NET 7 preview 1 and relnote the issue. 
Opened tracking issue to re-enable tests: https://github.com/PowerShell/PowerShell/issues/16883

Reply:
Once we update the master branch to use this flawed .NET version, it could cause problem to our development.
Also, what will the relnote say? `HttpClientHandler` may not be the only type that got affected, so are we going to call out that "resolution on some types may fail"?

Reply:
Will .Net team skip Preview1 too? I'd expect this if the bug is so unpredictable.